### PR TITLE
feat(curriculum): added more tests to file: rosetta code Tokenize a string with escaping

### LINKED ISSUE #54687: add more tests to rosetta code Tokenize a string with escaping
https://github.com/freeCodeCamp/freeCodeCamp/blob/dea35985c1490728a5a4dd5032ee22bae34b6ddd/curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md#L83

### PR DESCRIPTION
**Pull request by JLH13**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #54687

This file had a comment that it needed more test cases. I added in three more.

### CODE REVIEW

File: curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md [hunk 1]
```diff
@@ -80,9 +80,17 @@ assert.deepEqual(tokenize(testStr2, '&', '@'), res2);
 const testStr1 = 'one^|uno||three^^^^|four^^^|^cuatro|';
 const res1 = ['one|uno', '', 'three^^', 'four^|cuatro', ''];
 
-// TODO add more tests
 const testStr2 = 'a@&bcd&ef&&@@hi';
 const res2 = ['a&bcd', 'ef', '', '@hi'];
+
+const testStr3 = 'ab^cd^ef^^gh';
+const res3 = ['ab^cd^ef^gh'];
+
```
Comment (ilenia-magoni) [inline curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md:88]: you only added some strints and arrays, doing nothing with them, as such, no new test has been added.

**Review by majestic-owl448 — CHANGES_REQUESTED** [review]
Please properly add new tests. If you have questions you can reach out on Discord or the forum

**Comment by ojeytonwilliams** [comment]
Closing as stale.

File: curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md [hunk 2]
```diff
@@ -80,9 +80,17 @@ assert.deepEqual(tokenize(testStr2, '&', '@'), res2);
 const testStr1 = 'one^|uno||three^^^^|four^^^|^cuatro|';
 const res1 = ['one|uno', '', 'three^^', 'four^|cuatro', ''];
 
-// TODO add more tests
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/22-rosetta-code/rosetta-code-challenges/tokenize-a-string-with-escaping.md:83]: These tests all need to be reflected in the test cases for the user's code above.

**Pull request closed without merging**